.github/workflows/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+## Description
+<!-- Briefly describe the changes made in your pull request (PR) below. -->
+
+* 
+
+### Issue(s) fixed
+<!-- Include the issue number that this PR fixes. {Example: Fixes #123} -->
+
+Fixes #
+
+### Preview
+<!-- Provide a PR preview link to the page(s) changed. {Example: https://besu-docs-git-100-branch-hyperledger.vercel.app} -->
+
+* 
```

Comment:
```suggestion

<!-- Provide a PR preview link to the page(s) changed. {Example: https://besu-docs-git-100-branch-hyperledger.vercel.app} -->

```

Reply:
@alexandratran I think the spaces might make it a bit unclear where to add details. I've noticed in a couple of PRs that users are adding them directly under the heading. Might we please consider keeping it as it is?

Reply:
It is a standard [Markdown best practice](https://www.markdownguide.org/basic-syntax/#heading-best-practices) to include blank lines around headings, which we follow throughout our docs source code. IMO it should carry through to our PR templates for consistency, and since those html tags are comments that don't appear in the rendered PR description, it doesn't matter where people add details (after, before, or replacing the comment). Not a huge deal either way, but that's my two cents.

Reply:
okay. Applied suggestions